awx/main/models/workflow.py

```diff
@@ -659,6 +659,13 @@ def notification_data(self):
                 node_job_description = 'job #{0}, "{1}", which finished with status {2}.'.format(node.job.id, node.job.name, node.job.status)
             str_arr.append("- node #{0} spawns {1}".format(node.id, node_job_description))
         result['body'] = '\n'.join(str_arr)
+        result.update(
```

Comment:
`dict.update()` does take kwargs, so constructing another inner `dict` isn't necessary

Reply:
As an aside, I want to advocate for our team to get away from the style of using manual `dict()` calls in favor of `dict` literals.  It's less flexible.